client/blocks/jitm/index.jsx

```diff
@@ -176,11 +189,6 @@ JITM.propTypes = {
 	isFetching: PropTypes.bool,
 };
 
-JITM.defaultProps = {
-	template: 'default',
-	isFetching: false,
```

Comment:
This `isFetching` value doesn't need to be defaulted because the `isFetchingJITM` selector always returns a valid `boolean` value.